schemachange/sc_records.c

```diff
@@ -503,6 +503,7 @@ static int convert_record(struct convert_record_data *data)
              * a lock to the last page of the stripe.
              */
 
+            sleep(1);
```

Comment:
Remove this for this PR?

Reply:
Was testing lost insert -- will remove, thanks.